### DISCUSSION TITLE
totp 2fa not working

### DISCUSSION THREAD

**Discussion by nolash** [community/community#47809] [New to GitHub]
I am using python pytop for the 2fa. Recently it has stopped working, now the result is being rejected. I have time server running. what could be the cause?

**Comment by queenofcorgis** [comment]
If you're asking how to use the piece of software that you found in a repository on GitHub, the best way to contact the maintainers of that software is to:

- Check the README for instructions on how to operate the software and pointers to documentation or troubleshooting info.
- Check the [SUPPORT file](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-support-resources-to-your-project), if one exists, for instructions on how to best contact the maintainers for support.
- Check the [CONTRIBUTING guide](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/setting-guidelines-for-repository-contributors), if one exists. Sometimes if there isn't a SUPPORT file, the CONTRIBUTING guide will give instructions on how to contact the maintainers for support.

All of these documents can be found, if they exist, in the repository where you found the software itself.

**Reply by nolash** [reply]
no i am not. i am asking why the method i used to perform 2fa login to access my github account suddenly doesnt work anymore. I did not change the method. So the question becomes: did _you_ change something.
